deploy/iso/minikube-iso/board/minikube/aarch64/linux_aarch64_defconfig

```diff
@@ -1283,3 +1283,4 @@ CONFIG_IP_ADVANCED_ROUTER=y
 CONFIG_IP_ROUTE_MULTIPATH=y
 CONFIG_IP_MULTICAST=y
 CONFIG_IP_MULTIPLE_TABLES=y
+CONFIG_EXT4_FS_SECURITY=y
```

Comment:
we might have to enable 
CONFIG_SECURITY_FILE_CAPABILITIES
to be able to enable this

Reply:
Not needed, also not included with x86